apps/golden-sample-app/src/environments/environment.ts

```diff
@@ -55,8 +55,6 @@ export const authConfig: AuthConfig = {
   showDebugInformation: true,
 
   logoutUrl: document.location.origin + '/logout',
-  useSilentRefresh: true,
-  silentRefreshTimeout: 5000,
```

Comment:
This configuration was re-added, but (at least for me) causes automatic refresh to not work anymore. So that means API calls start failing after a couple of minutes.